sklearn/linear_model/bayes.py

```diff
@@ -216,10 +231,43 @@ def fit(self, X, y):
         self.alpha_ = alpha_
         self.lambda_ = lambda_
         self.coef_ = coef_
+        sigma_ = np.dot(Vh.T,
+                        Vh / (eigen_vals_ + lambda_ / alpha_)[:, None])
+        self.sigma_ = (1. / alpha_) * sigma_
 
         self._set_intercept(X_offset, y_offset, X_scale)
         return self
 
+    def predict(self, X, return_std=False):
+        """Predict using the linear model. In addition to the mean of the
```

Comment:
This is not pep 257: https://www.python.org/dev/peps/pep-0257/

Please add an empty line after the first sentence.